source/api/search/0.9/index.md

```diff
@@ -593,15 +603,31 @@ The example request above might generate the following response:
   "@type": "search:TermList",
   "ignored": ["box"],
   "terms": [
-    {"match": "bird", "count": 15},
-    {"match": "biro", "count": 3},
-    {"match": "birth", "count": 9},
-    {"match": "birthday", "count": 21}
+    {
+      "match": "bird", 
+      "url": "http://example.org/service/identifier/search?motivation=painting&q=bird",
+      "count": 15
+    },
+    {
+      "match": "biro", 
+      "url": "http://example.org/service/identifier/search?motivation=painting&q=biro",
+      "count": 3
+    },
+    {
+      "match": "birth", 
+      "url": "http://example.org/service/identifier/search?motivation=painting&q=birth",
+      "count": 9
+    },
+    {
+      "match": "birthday", 
+       "url": "http://example.org/service/identifier/search?motivation=painting&q=birthday",
+      "count": 21
+    }
   ]
 }
 {% endhighlight %}
 
-Semantic Tags or other URIs searchable via the `q` parameter might also have labels to display to the user, rather than the URI that matched.  The order of the terms might be by the label, rather than the matching URI, if there is an associated label.
+URIs or other data that are searchable via the `q` parameter might also have `label`s to display to the user, rather than the exact string that matched.  This can also be useful if stemming or other term normalization has occured, in order to display the original rather than the processed term.
 
```

Comment:
I get it, but 'might' sounds wishy-washy to me. I think a normal 'may' is the correct word.